h2/src/main/org/h2/jdbc/JdbcBlob.java

```diff
@@ -190,12 +190,11 @@ public OutputStream setBinaryStream(long pos) throws SQLException {
             if (value.getPrecision() != 0) {
                 throw DbException.getInvalidValueException("length", value.getPrecision());
             }
-            final JdbcConnection c = conn;
             final PipedInputStream in = new PipedInputStream();
             final Task task = new Task() {
                 @Override
                 public void call() {
-                    value = c.createBlob(in, -1);
+                    value = conn.createBlob(in, -1);
```

Comment:
This change generates an additional synthetic accessor method. Local final variable was used to avoid this.